nodebuilder/settings.go

```diff
@@ -83,7 +92,7 @@ func initializeMetrics(
 			exp,
 		),
 		controller.WithExporter(exp),
-		controller.WithCollectPeriod(2*time.Second),
+		controller.WithCollectPeriod(node.OptlCollectPeriodInSeconds*time.Second),
```

Comment:
collect period affects load on collector and a lot of other things, it should not be set based on one logic of one particular metric. Please remove

Reply:
It's the other way around, the metric is depending on this value. So we decided to put this hard-coded value into a constant so that it's accessible elsewhere.

Reply:
Lets discuss in another thread, since they are related